canonical-websites/cn.ubuntu.com
# Convert /news to v1 styles

**Note**: This branch is built on the changes in https://github.com/canonical-websites/cn.ubuntu.com/pull/151 - that PR should be landed before this one.

## Done

- Converted /news section of the site to Vanilla v1 styles


## QA

- Pull code
- Sanity check file deletions
- Run `./run serve --watch`
- View desktop page on http://0.0.0.0:8010/news
- Click through to news articles and check they look ok also

Fixes https://github.com/ubuntudesign/vanilla-squad/issues/70
